soroban-sdk-auth/src/public_types.rs

```diff
@@ -0,0 +1,58 @@
+use soroban_sdk::{contracttype, Bytes, BytesN, Env, EnvVal, Symbol, Vec};
+
+#[derive(Clone)]
+#[contracttype]
+pub struct KeyedEd25519Signature {
+    pub public_key: BytesN<32>,
+    pub signature: BytesN<64>,
+}
+
+pub type AccountAuthorization = Vec<KeyedEd25519Signature>;
+
+#[derive(Clone)]
+#[contracttype]
+pub struct KeyedAccountAuthorization {
+    pub public_key: BytesN<32>,
```

Comment:
Can we rename the public_key to account_id? In this context it is really just a identifier and we aren't afaik going to use it as a key. It helps to communicate that intent and as a bonus is consistent with language we use in the classic protocol.
```suggestion
    pub account_id: BytesN<32>,
```

Reply:
Done